modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/bluepyefefeatures/v0.1.2.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/bluepyefefeatures/v0.1.2/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:BluePyEfeFeaturesShape",
+      "@type": "sh:NodeShape",
+      "label": "BluePyEfe output shape",
+      "comment": "BluePyEfe extracted features.",
+      "targetClass": "nsg:BluePyEfeFeatures",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:configurationUsed",
```

Comment:
I suggest to use nsg:configuration as a property and have a its value be an entity of nsg:BluePyEfeConfiguration which has the actual configuration file attached instead of:

>Direct link to the EFE configuration which was used to produce these features

Reply:
Removed, this link will be discoverable through activity.